spec/factories/participant_groups.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :participant_group do
+    event_id 1
+    user_id 1
```

Comment:
This could lead to problems if Events or Users with that id don't exist, or if User 1 isn't actually a participant for Event 1.